### ISSUE TITLE
[BUG] IgnoreKeys can't handle Backspace

### ISSUE THREAD

**Issue by AndriySvyryd** [greena13/react-hotkeys#172]
When I use
```JSX
<IgnoreKeys only={[]} except={["Backspace"]}>
…
</IgnoreKeys>
```

I get this in the console:
> 0: Backspace withHotKeysIgnoreOverride.js (177,6)

Version:
"react-hotkeys": "2.0.0-pre4"

**Comment by AndriySvyryd** [comment]
@greena13 No, `"BackSpace"`, `"backspace"` and `"backSpace"` behave the same

**Comment by greena13** [comment]
Fix is now available in [v2.0.0-pre6](https://github.com/greena13/react-hotkeys/releases/tag/v2.0.0-pre6).
